v3/src/components/graph/components/graph.tsx

```diff
@@ -103,14 +110,13 @@ export const Graph = observer((
 
     // MouseOver events, if over an element, brings up hover text
     function showDataTip(event: MouseEvent) {
-      const
-        target = select(event.target as SVGSVGElement)
+      const target = select(event.target as SVGSVGElement)
       if (target.node()?.nodeName === 'circle' && dataset) {
         target.transition().duration(transitionDuration).attr('r', hoverPointRadius)
         const [, caseID] = target.property('id').split("_"),
           attrIDs = graphModel.config.uniqueTipAttributes,
           tipText = getPointTipText(dataset, caseID, attrIDs)
-        tipText !== '' && dataTip.show(tipText, event.target)
+          tipText !== '' && dataTip.show(tipText, event.target)
```

Comment:
This indent is incorrect -- it should be at the same level as the `const` (as it was) since it's not one of the values being defined by the `const`.

Reply:
Oh, thanks, I see that now.  I'll move it back in my next branch.